backend/src/test/java/com/github/boltzmann/biokiste/backend/controller/PostBoxWithNewUserTest.java

```diff
@@ -0,0 +1,125 @@
+package com.github.boltzmann.biokiste.backend.controller;
+
+import com.github.boltzmann.biokiste.backend.model.OrganicBox;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+class PostBoxWithNewUserTest extends CrudTestWithLogIn{
+
+    private final OrganicBox box = OrganicBox.builder()
+            .id("7")
+            .customers(List.of("4711"))
+            .build();
+
+    @Test
+    void postBox_whenOtherValidUserIsAddedToBox_then200AndGetUpdatedBox(){
+        // Given
+        createTestUserInLoginRepoAndGet("-1", "Other User" , "Pass the wort!");
+        String jwt = getTokenFor("Other User", "Pass the wort!");
+        organicBoxRepository.insert(box);
+        // When
+        OrganicBox actual = getPostedOrganicBox(jwt, "7");
+        // Then
+        OrganicBox expected = OrganicBox.builder()
+                .id("7")
+                .customers(List.of("4711", "-1"))
+                .build();
+        Assertions.assertEquals(expected, actual);
+    }
+
+    @Test
+    void PostBox_whenValidUserIsAddedToBox_then200AndGetUpdatedBox(){
+        // Given
+        createTestUserInLoginRepoAndGet("42", "Test User" , "Pass the wort!");
+        String jwt = getTokenFor("Test User", "Pass the wort!");
+        organicBoxRepository.insert(box);
+        // When
+        OrganicBox actual = getPostedOrganicBox(jwt, "7");
+        // Then
+        OrganicBox expected = OrganicBox.builder()
+                .id("7")
+                .customers(List.of("4711", "42"))
+                .build();
+        Assertions.assertEquals(expected, actual);
+    }
+
+    @Test
+    void postBox_whenThereIsNoCustomerInBox_AddNewCustomerWith200(){
+        // Given
+        createTestUserInLoginRepoAndGet("23", "New User" , "Pass the wort!");
+        String jwt = getTokenFor("New User", "Pass the wort!");
+        // "A number  $n$  is called lonely if its distance to closest prime sets a new record."
```

Comment:
sweet but irrelevant?

Reply:
Well...one can understand the test better. Will delete it.